packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -812,6 +828,7 @@ export function TransferPanel() {
             destinationAddress,
             txLifecycle: {
               onTxSubmit: () => {
+                setTransactionHistoryTab(TransactionHistoryTab.DEPOSITS)
```

Comment:
NIT: There are 2 more instances where we open `openTransactionHistoryPanel` (ie. in Header and in Transaction status bar) but we are not setting the tab there so it will revert to the last opened tab state in that case, is that expected?

Reply:
Good point, I have changed it to be consistent with what we have on production (open on deposit tab by default)